<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tkarded</class>
 <widget class="QMainWindow" name="tkarded">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
border: 3px solid rgb(0, 0, 127);
background-color: rgb(170, 170, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QLabel{
border: none;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QComboBox{
border: none;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}
QLineEdit{
border: none;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;}

QPushButton{
border: 1px solid rgb(0, 0, 127);
border-radius:10px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(170, 85, 255);
}

QPushButton:hover{
background-color: rgb(170, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
border: none;
	background-color: rgb(158, 158, 158);
}</string>
         </property>
         <layout class="QHBoxLayout" name="layout_principal">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_title">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>T KARDEX MATERIAL</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ampliar">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(0, 0, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border:none;
border-radius:0px;
}

QPushButton:hover{
	background-color: rgb(170, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>â– </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border:none;
border-radius:0px;
}

QPushButton:hover{
	background-color: rgb(170, 0, 255);
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>31</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Seleccione</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Codigo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nombre</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>MEDIDA : </string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QTableWidget" name="table_qwk_new">
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(176, 183, 232, 255));</string>
           </property>
           <property name="autoScrollMargin">
            <number>16</number>
           </property>
           <column>
            <property name="text">
             <string>Codigo proyecto</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Fecha</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Entradas</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Salidas</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Devoluciones</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Saldo</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo de trabajo</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Lote</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Responsable</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Observaciones</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Firma</string>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
